client/express-checkout/utils/index.ts

```diff
@@ -169,18 +169,31 @@ export const displayLoginConfirmation = (
 	}
 };
 
+type ButtonAttributesType =
+	| { height: string; borderRadius: string; darkMode: boolean }
+	| undefined;
+
 /**
  * Returns the appearance settings for the Express Checkout buttons.
  * Currently only configures border radius for the buttons.
  */
-export const getExpressCheckoutButtonAppearance = () => {
+export const getExpressCheckoutButtonAppearance = (
+	buttonAttributes: ButtonAttributesType
+) => {
+	let borderRadius = getDefaultBorderRadius();
```

Comment:
For me this generates an error locally, saying `wcpaySettings` is `undefined`. Not quite sure where to go with it, seems like it should be exported as a global but it's not there 🤷

Reply:
Yeah, I'm seeing the same thing. It started happening as soon as you added `getDefaultBorderRadius()` there in 2abb118e4d500dd25630b79214677f3025043b53 and I think it's uncovering a bad assumption from when we added the utility function.

It seems `wcpaySettings` isn't always loaded (like we expected while adding support for ECE) but we never got an issue because the previous version of `getExpressCheckoutButtonAppearance` never actually called `getDefaultBorderRadius`. The border radius was always present on `wcpayExpressCheckoutParams.button`.

I pushed a fix for this in dcdb55c0d55730cf440f4c8eb620ee6be16484c5 👍